class-wc-calypso-bridge-shared.php

```diff
@@ -111,7 +111,7 @@ public function add_scripts() {
 		$params      = array(
 			'isEcommercePlan'              => (bool) wc_calypso_bridge_has_ecommerce_features(),
 			'isEcommercePlanTrial'         => (bool) wc_calypso_bridge_is_ecommerce_trial_plan(), // This is true for ecommerce trial only.
-			'isWooNavigationEnabled'       => (bool) apply_filters( 'ecommerce_new_woo_atomic_navigation_enabled', 'yes' === get_option( 'wooexpress_navigation_enabled', 'yes' ) ),
+			'isWooNavigationEnabled'       => (bool) apply_filters( 'ecommerce_new_woo_atomic_navigation_enabled', 'yes' === get_option( 'wooexpress_navigation_enabled', 'yes' ) ) && \Jetpack::is_module_active( 'sso' ),
```

Comment:
We should check for existence of `Jetpack` class

Reply:
Thanks for the review, @ilyasfoo! There are additional instances where the Jetpack class is presumed, but I have implemented defensive checks in this section. We can continue to evaluate any remaining assumptions as we move forward.